synchronization/src/cyface/java/de/cyface/synchronization/CyfaceSyncAdapter.java

```diff
@@ -217,26 +235,48 @@ private void syncMeasurement(final String authority, final ContentProviderClient
             // We delete the points in each iteration so we load always from cursor position 0
             final JSONObject measurementSlice = fillMeasurementSlice(measurementSliceTemplate, dataAccessLayer, 0, 0, 0,
                     0, geoLocationJsonMapper, accelerationJsonMapper, rotationJsonMapper, directionJsonMapper);
-            postMeasurementSlice(syncResult, context, url, account, measurementSlice);
+            final boolean transmissionSuccessful = postMeasurementSlice(syncResult, context, url, account,
+                    measurementSlice);
+            if (!transmissionSuccessful) {
+                return false;
+            }
             try {
                 // TODO: This way of deleting points is probably rather slow when lots of data is
                 // stored on old devices (from experience). We had a faster but uglier workaround
                 // but won't reimplement this here in the SDK. Instead we'll use the Cyface Byte Format
                 // from the Movebis flavor and the file uploader which we'll implement before releasing this.
                 // TODO: We should probably remove the unused isSynced flag from the points after
-                // we implemented the Cyface Byte Format synchronization. #
+                // we implemented the Cyface Byte Format synchronization. #CY-3592
 
                 // We delete the data of each point type separately to avoid #CY-3859 parcel size error.
                 deletePointsOfType(provider, authority, measurementSlice, geoLocationJsonMapper);
                 deletePointsOfType(provider, authority, measurementSlice, accelerationJsonMapper);
                 deletePointsOfType(provider, authority, measurementSlice, rotationJsonMapper);
                 deletePointsOfType(provider, authority, measurementSlice, directionJsonMapper);
```

Comment:
Thoase 4 things should happen in loader.cleanMeasurement. There are two implementations of this method. The one for Movebis just deletes accelerations, rotations and directions, while the one for Cyface deletes the measurement.

Reply:
The reason why we separately delete points here is:
- We still transmit measurements in slices, i.e. we want to delete the points after each successful slice transmission. This way it's no problem if the transmission is interrupted after a few slices.
- Only when a measurement was fully transmitted, we use the loader.cleanMeasurement method to drop everything related with this measurement (the measurement entry itself)